lib/ecto/association.ex

```diff
@@ -295,9 +295,10 @@ defmodule Ecto.Association.Has do
 
   @doc false
   def build(%{assoc: assoc, owner_key: owner_key, assoc_key: assoc_key,
-              queryable: queryable}, struct) do
+              queryable: queryable}, struct, attributes \\ %{}) do
```

Comment:
We don't need to have the default argument in here because this is always invoked as a callback.